package.json

```diff
@@ -22,7 +22,7 @@
     "should": "^13.0.0"
   },
   "engines": {
-    "node": ">=0.10"
+    "node": ">=6.0"
```

Comment:
This needs to be released with change in the major version number.

Reply:
Works for me, we're likely going to deprecate some methods we can't support with an upcoming N-API implementation from #189.

We should put deprecation warnings in for each of the methods, can you make a PR with the ones we should stop supporting @rolftimmermans?

Reply:
I think technically we can make this `node": ">=5.0` since prebuild only dropped support for Node 4.

Reply:
That's great that prebuild can support further back! Since the N-API changes coming up are going to force us into requiring 8.x+, I'd like us to push a bit further forward (old releases of ours will still work for those wanting to stick around on old node.js version). Aligning with 6.x puts us in sync with the [node LTS schedule](https://github.com/nodejs/Release)